Index: product/pitches/Basic Funding Dashboard.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/product/pitches/Basic Funding Dashboard.md b/product/pitches/Basic Funding Dashboard.md
--- a/product/pitches/Basic Funding Dashboard.md	(revision 16ac092677aa9ac9c26ea62ed2feb9793432eff)
+++ b/product/pitches/Basic Funding Dashboard.md	(revision 85026407876f683ff8a50b908e9de53fecfb9e4f)
@@ -21,7 +21,7 @@
 
 ## Proposal
 
-This proposal is an MVP created as a low cost (and correspondingly, at least initially, moderate impact) tool **based primarily on data that is already available on the platform**. It does however serve as a gravitational center that will expand as we advance on the OSC product roadmap. Ultimately it is designed to become a tool where funders can formulate, execute and monitor the impact of their funding strategy.
+This proposal is an MVP created as a low cost (and correspondingly, at least initially, moderate impact) tool **based primarily on data that is already available on the platform**. It does however serve as a gravitational center that will expand as we advance on the OSC product roadmap. Ultimately it is designed to become a tool where funders can formulate, execute and monitor the impact of their funding strategy.  
 
 ### Basic Dashboard
 
